vipul/Gallery_System/fashion.html

```diff
@@ -0,0 +1,105 @@
+							<li><a href="portrait.html">Portrait</a></li>
+							<li><a href="fashion.html">Fashion</a></li>
+							<li><a href="macro.html">Macro</a></li>
+							<li><a href="bird.html">Bird</a></li>
+							<li><a href="nature.html">Nature</a></li>
+							<li><a href="food.html">Food</a></li>
+						</ul>
+					</li>
+					<li><a href="services.html">SERVICES</a></li>
+					<li><a href="about.html">ABOUT</a></li>
+					<li><a href="contact.html">CONTACT</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="category-wild">
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion.jpg">
+						<img src="images/fashion.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion1.jpg">
+						<img src="images/fashion1.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion2.jpg">
+						<img src="images/fashion2.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion3.jpg">
+						<img src="images/fashion3.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion4.jpg">
+						<img src="images/fashion4.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion5.jpg">
+						<img src="images/fashion5.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion6.jpg">
+						<img src="images/fashion6.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion7.jpg">
+						<img src="images/fashion7.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div><div class="gallery-wild">
+					<a target="_blank" href="images/fashion8.jpg">
+						<img src="images/fashion8.jpg" alt="Picture of Fashion Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion9.jpg">
+						<img src="images/fashion9.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion.jpg">
+						<img src="images/fashion.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/fashion1.jpg">
+						<img src="images/fashion1.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+		</div>
+		<div class="footer">
+			<p>
+				Copyright 2020 All rights reserved | This is developed by <a href="https://www.facebook.com/vipul.suryawanshi" target="_blank">Vipul Suryawanshi.</a>
+			</p>
+		</div>
+	</body>	
+</html>	
```

Comment:
remove all extra spaces
Add more content in footer
Add contact us in footer
bad code indentation

Reply:
i will do all the changes.